Pancake_Sort/Pancake_Sort.dart

```diff
@@ -0,0 +1,70 @@
+/**
+ *   Pancake Sort
+ * --------------------------
+ * It is a sorting algorithm of O(n^2) algorithm which sorts like selection sort
+ * by moving the minimum to the front by reversing the array till its index.
+ */
+// Importing required libraries
+import 'dart:io';
+
+// Utility function to find maximum of the array
+int findMax(List array, int n){
+    int max_value = array[0];
+    int max_index = 0;
+    for(int i = 0; i < n; i ++){
+        if(max_value < array[i]){
+            max_value = array[i];
+            max_index = i;
+        }
+    }
+    return max_index;
+}
+
+// Utility function to flip an array from an index
+void flip(List array, int i){
+    int start = 0, temp;
+    while(start < i){
+        temp = array[start];
+        array[start] = array[i];
+        array[i] = temp;
+        start ++;
+        i --;
+    }
+}
+
+// Function to sort a list using pancake sort
+void pancakeSort(List array, int n){
+    for(int i = n; i > 1; i --){
+        int max_index = findMax(array, i);
+        if(max_index != i - 1){
+            flip(array, max_index);
+            flip(array, i - 1);
+        }
+    }
+}
+// Driver method of the program
```

Comment:
give one blank line before this line. and subsequent blank lines are not needed after the comment.